doc/en/components/grids/_shared/filtering.md

```diff
@@ -342,7 +344,58 @@ constructor() {
 }
 ```
 
-<!-- end: Angular, WebComponents -->
+```razor
+<{ComponentSelector}
+    FilteringExpressionsTree="filteringExpressions"
+    AllowFiltering="true"
+    FilterMode="FilterMode.QuickFilter"
+    ...
+
+@code {
+
+    protected override async Task OnInitializedAsync()
+    {
+        IgbFilteringExpressionsTree gridFilteringExpressionsTree = new IgbFilteringExpressionsTree() { Operator = FilteringLogic.And };
+        IgbFilteringExpression productFilteringExpressionsTree = new IgbFilteringExpression()
+            {
+            FieldName = "Category",
+            ConditionName = "contains",
+            IgnoreCase = true,
+            SearchVal = "Metals"
+        };
+    
+        gridFilteringExpressionsTree.FilteringOperands = new IgbFilteringExpression[1] { productFilteringExpressionsTree };
+        this.filteringExpressions = gridFilteringExpressionsTree;
+    }
+
+    public IgbFilteringExpressionsTree filteringExpressions;
+}
+```
+
+```tsx
+private filteringExpressions: IgrFilteringExpressionsTree;
+
+constructor(props: any) {
+    super(props);
+
+    const gridFilteringExpressionsTree = { operator: FilteringLogic.And } as IgrFilteringExpressionsTree;
+    const productFilteringExpressionsTree = { 
+        fieldName: "ProductName",
+        conditionName: "contains",
+        ignoreCase: true,
+        searchVal: "Chai"
+    } as IgrFilteringExpression;
+
+    gridFilteringExpressionsTree.filteringOperands = [ productFilteringExpressionsTree ];
+    this.filteringExpressions = gridFilteringExpressionsTree;
+}
+
+public render(): JSX.Element {
+    return ({ComponentSelector}
```

Comment:
Can you close the render object? Even if it is not complete, better be valid.
Also the `{ComponentSelector}` needs to manually include the `<` and later closeing `>` symbols.